index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header class="navbar">
+      <a href="#">Dan Logo</a>
+      <span> <img src="images/burger.png" alt="toggle icon" /> </span>
+    </header>
+    <section class="header">
+      <div class="header-title">
+        <h1>Hey there. I'm Dan Elifeleti</h1>
+        <h2 class="green">I'm a software developer</h2>
+      </div>
+      <div class="header-content">
+        <p>
+          I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a
+          project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="icon-style">
+        <ul class="icons">
```

Comment:
- [ ] kindly add a little space between your icons so they follow the figma design.
yours
![Screenshot (195)_LI](https://user-images.githubusercontent.com/65298048/176601779-73fe7296-a170-4dd0-9de4-8f8d03e77986.jpg)
figma
![Screenshot (194)](https://user-images.githubusercontent.com/65298048/176601901-88d219db-7b6a-42ea-93b6-b45599c3dff2.png)

Reply:
ok